### PR TITLE
Check HTTPS

### PR DESCRIPTION
**Pull request by IoyoCode**
[contributing]: https://github.com/Negative-Games/Framework/blob/main/CONTRIBUTING.md

## Pull Request Etiquette 

 
- [x] I have checked the PRs for upcoming features/bug fixes. 
- [x] I have read the [contributing guidelines][contributing]. 

 
### Changes 
- [x] Internal code 
- [ ] Library interface (affecting end-user code) 
- [ ] Documentation 
- [ ] Other: \_____  

 

Closes Issue: NaN 

## Description 
Checking for "https://www" & "https://" and adapting.

### CODE REVIEW

**Comment by IoyoCode** [comment]
branch please

File: src/main/java/games/negative/framework/util/HasteBin.java [hunk 1]
```diff
@@ -47,14 +47,29 @@
...
 public class HasteBin {
+    String binURL = "https://bin.hypews.com/";
+
+    /**
+     * Create a new HasteBin instance with your custom HasteBin (optional)
+     *
+     * @author Seailz
+     * @param binURL The exact URL to your pastebin - for example https://bin.seailz.com/
+     */
+    public HasteBin(@NotNull String binURL) {
```
Comment (joeecodes) [inline src/main/java/games/negative/framework/util/HasteBin.java:58]: Screenshot 2022-05-08 at 16 37 35
Screenshot 2022-05-08 at 16 38 24

this simply wont work

Comment (joeecodes) [reply src/main/java/games/negative/framework/util/HasteBin.java:58]: Please make sure this code works.

Comment (joeecodes) [reply src/main/java/games/negative/framework/util/HasteBin.java:58]: There also needs to be a default link (bin.hypews.com) incase the end user does not have their own hastebin..

**Comment by IoyoCode** [comment]
can i have a branch

**Comment by joeecodes** [comment]
> can i have a branch

you have your local one

**Review by joeecodes — CHANGES_REQUESTED** [review]

**Review by joeecodes — CHANGES_REQUESTED** [review]
please review comments on hastebin.java

File: src/main/java/games/negative/framework/util/HasteBin.java (hunk 1 above)
Comment (IoyoCode) [reply src/main/java/games/negative/framework/util/HasteBin.java:58]: > There also needs to be a default link (bin.hypews.com) incase the end user does not have their own hastebin..

That's already a thing..

**Comment by IoyoCode** [comment]
I'll do this in a bit, hanging out with friends atm

File: src/main/java/games/negative/framework/util/HasteBin.java (hunk 1 above)
Comment (joeecodes) [reply src/main/java/games/negative/framework/util/HasteBin.java:58]: ah ye im stipid

**Comment by IoyoCode** [comment]
actually just gonna close this, not really required and the end user can just deal with the error themselves ig

**Comment by joeecodes** [comment]
> actually just gonna close this, not really required and the end user can just deal with the error themselves ig

thats not how it works. the issue is at the code base, not with how the user uses the API. Either fix it or this feature will not be added.

**Comment by IoyoCode** [comment]
I'm saying it **shouldn't be** implement, sorry if I wasn't clear.

**Pull request closed without merging**